### ISSUE TITLE
Balance crafted items EMC values to avoid transmutation table playthroughs

### ISSUE THREAD

**Issue by LouVanSoom** [xJon/Tekkit-2#256]
### Was this feature suggested before?

- [X] No, it was not.

### What is your suggestion?

Make crafted items have a higher EMC value when pulling them OUT of transmutation table so you have to 'pay' EMC for each crafting stage so to speak to stop transmutation table abuse (entire modpack almost gets reduced to craft 1 item and get it out of table for the rest of the playthrough which is really boring imo). When putting the crafted items IN the table they should provide EMC value equal to their constituent parts to avoid infinite EMC abuse as well then. I feel like this is a good alternative to setting a 0.x% transmutation penalty or perhaps even to be used in conjunction with it as ProjectE is very reductive to the automation aspect of Tekkit I feel like.

**Comment by xJon** [comment]
Keep in mind a lot of items don't have an EMC value.

**Comment by LouVanSoom** [comment]
Yeah didn't really think that through, was transmuting a bunch of pistons and assumed a lot of other items would have that issue.
